packages/dashboard/src/components/TfChainConnector.vue

```diff
@@ -225,6 +225,7 @@ import { Component, Vue, Watch } from "vue-property-decorator";
 import { generateKeyPair } from "web-ssh-keygen";
 
 import config from "@/portal/config";
+import { getBalance } from "@/portal/lib/balance";
```

Comment:
I see you imported this function and it's never used, so remove it please if not needed.